### PR TITLE
fix bootstrap version working correctory

### PR DESCRIPTION
**Pull request by ainoya**
Related issue: https://github.com/yeoman/generator-angular/issues/1116

### CODE REVIEW

**Comment by mookjp** [comment]
I'm not sure what kind of problem you met. You couldn't make any refs to bootstrap in built static files, right?

File: docker/pool/build-screen/bower.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "name": "build-screen",
   "private": true,
   "dependencies": {
-    "bootstrap": ">=3.3.0"
+    "bootstrap": "=3.3.4"
```
Comment (mookjp) [inline docker/pool/build-screen/bower.json]: I agree with you to use newer version of libraries as some features/fixes are merged, so what do you think to use `>=` version semantics?
If problem you met is came from any version except for `3.3.4`, I agree to use fixed version.

**Comment by ainoya** [comment]
> I'm not sure what kind of problem you met. You couldn't make any refs to bootstrap in built static files, right?

version 3.3.5 isn't working correctly. I think this kind of version should be specified by fixed version as possible as we can. ( if `bootstrap.css` is major upgraded didn't have backward compatibility, this build-screen's looks may be potentially broken.)

**Comment by mookjp** [comment]
I understand. Thanks!

**Pull request merged**